examples/lorenz_attractor.ipynb

```diff
@@ -10,6 +10,12 @@
     "\n",
     "If you are looking at a static version of this notebook and would like to run its contents, head over to [GitHub](https://github.com/giotto-ai/giotto-tda/blob/master/examples/lorenz_attractor.ipynb) and download the source.\n",
     "\n",
+    "## See also\n",
+    "\n",
+    "- [Topology of time series example](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html), in which the *Takens embedding* technique is explained on simple examples and in detail.\n",
+    "- [Gravitational waves detection](https://giotto-ai.github.io/gtda-docs/latest/notebooks/gravitational_waves_detection.html), where the *Takens embedding* technique helps us find recurrent structures in noisy signals from gravitational waves, and detect their presence.\n",
```

Comment:
This is maybe a bit much but since the notebook is an illustration of other people's work, we can avoid using "us", e.g. `where, following [arXiv:1910.08245](https://arxiv.org/abs/1910.08245), the Takens embedding technique is shown to be effective for the detection of gravitational waves signals buried in background noise.`

(no need for italics because Takens embedding has been mentioned above)